Framework/Core/test/test_TreeToTable.cxx

```diff
@@ -75,26 +88,32 @@ BOOST_AUTO_TEST_CASE(TreeToTableConversion)
   for (int ii = 0; ii < table->num_rows(); ii++) {
     ntrueout += oks->Value(ii) ? 1 : 0;
   }
+  
 
   // test result
   BOOST_REQUIRE_EQUAL(table->Validate().ok(), true);
   BOOST_REQUIRE_EQUAL(table->num_rows(), ndp);
-  BOOST_REQUIRE_EQUAL(table->num_columns(), 6);
+  BOOST_REQUIRE_EQUAL(table->num_columns(), ncols);
   BOOST_REQUIRE_EQUAL(table->column(0)->type()->id(), arrow::boolean()->id());
   BOOST_REQUIRE_EQUAL(ntruein, ntrueout);
   BOOST_REQUIRE_EQUAL(table->column(1)->type()->id(), arrow::float32()->id());
   BOOST_REQUIRE_EQUAL(table->column(2)->type()->id(), arrow::float32()->id());
   BOOST_REQUIRE_EQUAL(table->column(3)->type()->id(), arrow::float32()->id());
   BOOST_REQUIRE_EQUAL(table->column(4)->type()->id(), arrow::float64()->id());
   BOOST_REQUIRE_EQUAL(table->column(5)->type()->id(), arrow::int32()->id());
+  BOOST_REQUIRE_EQUAL(table->column(6)->type()->id(), arrow::fixed_size_list(arrow::float64(),nelem)->id());
 
   f1.Close();
 
+  printf("Write table to tree ...\n");
   TFile* f2 = new TFile("table2tree.root", "RECREATE");
   TableToTree ta2tr(table, f2, "mytree");
   stat = ta2tr.addAllBranches();
   auto t2 = ta2tr.process();
+  
+  t2->Print();
```

Comment:
Try not to have any printout for the tests. If it's to check the content by eye, you should probably check it with BOOST_REQUIRE_EQUAL or similar.

Reply:
needs to be cleaned up